packages/atomic/src/components/search-box-suggestions/atomic-search-box-recent-queries/atomic-search-box-recent-queries.tsx

```diff
@@ -122,16 +122,18 @@ export class AtomicSearchBoxRecentQueries {
       key: 'recent-query-clear',
       content: (
         <div class="flex justify-between w-full">
-          <span class="font-bold">
+          <span class="font-bold" part="recent-query-title">
             {this.bindings.i18n.t('recent-searches')}
           </span>
-          <span>{this.bindings.i18n.t('clear')}</span>
+          <span part="recent-query-clear">{this.bindings.i18n.t('clear')}</span>
         </div>
       ),
       onSelect: () => {
         this.recentQueriesList.clear();
         this.bindings.triggerSuggestions();
       },
+      part: 'recent-query-header',
```

Comment:
Could that part be set on the div at line 124? It would remove the necessity for the prop

Reply:
If people want to hide the clear button, I think it should be done with a prop on `atomic-search-box-recent-queries`. I wouldn't try to support either method until there's demand for it though.